### PR TITLE
fixed broken owner test

### PR DESCRIPTION
**Pull request by dabasov**

### CODE REVIEW

File: tests/cli_tests/zwalletcli_storage_update_config_test.go [hunk 1]
```diff
@@ -172,3 +172,21 @@ func updateStorageSCConfig(t *testing.T, walletName string, param map[string]int
 		return cliutils.RunCommandWithoutRetry(cmd)
 	}
 }
+
+func updateStorageSCConfigWithNonce(t *testing.T, walletName string, param map[string]interface{}, nonce int64, retry bool) ([]string, error) {
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zwalletcli_storage_update_config_test.go:176]: This is great, but we should remove ` updateStorageSCConfig` entirely and implement nonce check in storage config tests too.

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

File: tests/cli_tests/config/zbox_config.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-block_worker: https://dev.0chain.net/dns
+block_worker: https://dev-2.devnet-0chain.net//dns
```
[the remark this answers is not in the document]
Comment (Kishan-Dhakan) [reply tests/cli_tests/config/zbox_config.yaml]: fixed

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/broken_owner_test  |
| gosdk  | staging  |
| zboxcli  | fix/get_config  |
| zwalletcli  | fix/get_config  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/broken_owner_test  |
| gosdk  | staging  |
| zboxcli  | fix/get_config  |
| zwalletcli  | fix/get_config  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

File: tests/cli_tests/zwalletcli_storage_update_config_test.go [hunk 3]
```diff
@@ -172,3 +161,21 @@ func updateStorageSCConfig(t *testing.T, walletName string, param map[string]int
 		return cliutils.RunCommandWithoutRetry(cmd)
 	}
 }
+
+// func updateStorageSCConfig(t *testing.T, walletName string, param map[string]interface{}, nonce int64, retry bool) ([]string, error) {
```
Comment (stewartie4) [inline tests/cli_tests/zwalletcli_storage_update_config_test.go:165]: remove this commented out code

**Pull request merged**